webapp/views.py

```diff
@@ -75,8 +67,10 @@ def post(self, request):
             if not account.exists():
                 raise Http404 ("The requested Wallet was not found.")
             else:
-                new_wallet = Wallet(wallet_number = received_json_data['wallet_number'],
-                                    account_number = received_json_data['account_number'])
+                new_wallet = Wallet(account_number=received_json_data['account_number'],
+                                    wallet_number=received_json_data['wallet_number'],
+                                    beneficiary_id=received_json_data.get('beneficiary_id', None),
```

Comment:
Beneficiary should not be provided at creation time, so no  need of this

Reply:
the change made by observation